app/views/content/landing/class-of-now/_article.erb

```diff
@@ -0,0 +1,67 @@
+<div class="row inset">
+  <section class="col col-640 col-center associated-logos">
+    <p><strong>In partnership with</strong></p>
+
+    <div>
+      <%= image_pack_tag("media/images/content/class-of-now/vice_logo.svg", alt: "Vice logo") %>
+      <%= image_pack_tag("media/images/content/class-of-now/refinery29_logo.svg", alt: "Refinery29 logo") %>
+      <%= image_pack_tag("media/images/content/class-of-now/dfe_logo.svg", alt: "Department for Education logo") %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <p>We've teamed up with Vice and Refinery29 to celebrate the different and dynamic faces of teaching today.</p>
+
+    <p>If you're looking for a career that allows you to bring your passions and personality to life, then teaching could be for you.</p>
+  </section>
+
+  <section class="col col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Inspire future generations",
+      colour: "pink",
+      image_1_path: "media/images/content/class-of-now/collage-1.jpg",
+      image_1_alt: "A teacher standing outside a classroom",
+      image_2_path: "media/images/content/class-of-now/collage-2.jpg",
+      image_2_alt: "A teacher sitting in front of lockers",
+    ) do %>
+      <p>
+        Make a difference to young people's lives and help them to see the world differently.
+      </p>
+    <% end %>
+  </section>
+
+  <section class="col col-720">
+    <h3>Everyday is different</h3>
+
+    <p>Life as a teacher means constantly being pleasantly surprised by your students and their perspectives.</p>
+
+    <h3>Grow as a person</h3>
+
+    <p>Teaching can help you build your confidence and develop interpersonal skills, which will prepare you for wherever your career takes you.</p>
+
+    <h3>Get rewarded for your work</h3>
+
+    <p>A starting salary of at least £28,000, or higher if you're working in London and opportunities to progress and develop your professional skills.</p>
+   
+   <h3> Help to get you into the classroom </h3>
+   
+    <ul>
+      <li>
+        <strong>Funding to help you train</strong> - learn more about
+        <%= link_to("the options available", "/funding-and-support/") %>
+      </li>
+      <li>
+        <strong>A helping hand</strong>- our advisers can offer free <%= link_to("one-to-one advice", "/teacher-training-advisers") %>
```

Comment:
```suggestion
        <strong>A helping hand</strong> - our advisers can offer free <%= link_to("one-to-one support", "/teacher-training-advisers") %>
```
@philspray 

Missing space - plus would it be better to say support here rather than advice, given we've already said `advisers`? But very happy for you to disagree as this is very minor!